ic-os/components/boundary-guestos/opt/ic/bin/bootstrap-ic-node.sh

```diff
@@ -68,6 +68,7 @@ function process_bootstrap() {
         ic_boundary.conf
         canister-ratelimit.yml
         network.conf
+        remuneration.conf
```

Comment:
Why do we need this in the boundary guestos? The boundary guestos is not NNS managed.